Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py

```diff
@@ -342,6 +344,9 @@ def parse_args(args):
         optparse.make_option(
             "--use-gpu-process", action="store_true", default=False,
             help=("Enable all GPU process related features, also set additional expectations and the result report flavor.")),
+        optparse.make_option(
+            "--no-use-gpu-process", action="store_true", default=False,
+            help=("Disable GPU process for DOM rendering.")),
```

Comment:
Similar to PR #12222, we should consider `arguments.NoAction` here:
```
from webkitcorepy import arguments
...
optparse.make_option(
    '--use-gpu-process', '--no-use-gpu-process',
    dest='use_gpu_process', default=None,
    help='...',
    action=arguments.NoAction,
)
```
Here I think the case for `arguments.NoAction` is much stronger because it looks like we convert these to a feature flag. That would mean `use_gpu_process =True -> UseGPUProcessForDOMRenderingEnabled` and `use_gpu_process =False -> UseGPUProcessForDOMRenderingEnabled=0` and `use_gpu_process =None` would pass nothing (which would have a GPU process enabled in WebKit but disabled in WebKitLegacy if I'm understanding the code correctly?). It does mean that something like `--no-gpu-process` and `--dump-render-tree` is redundant, but I think that's true in the current implementation anyways.

Reply:
It doesn't seem to work?

```
% ./Tools/Scripts/run-webkit-tests --debug --no-build --no-show-results fast/scrolling/scroll-animator-basic-events.htm
Traceback (most recent call last):
  File "/Volumes/Data/safari-2/OpenSource/./Tools/Scripts/run-webkit-tests", line 46, in <module>
    sys.exit(main(sys.argv[1:], sys.stdout, sys.stderr))
  File "/Volumes/Data/safari-2/OpenSource/Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py", line 57, in main
    options, args = parse_args(argv)
  File "/Volumes/Data/safari-2/OpenSource/Tools/Scripts/webkitpy/layout_tests/run_webkit_tests.py", line 123, in parse_args
    optparse.make_option("--remote-layer-tree", "--no-remote-layer-tree", dest="remote_layer_tree", action=arguments.NoAction, default=None,
  File "/AppleInternal/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/optparse.py", line 581, in __init__
    checker(self)
  File "/AppleInternal/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/optparse.py", line 636, in _check_action
    raise OptionError("invalid action: %r" % self.action, self)
optparse.OptionError: option --remote-layer-tree/--no-remote-layer-tree: invalid action: <class 'webkitcorepy.arguments.NoAction'>
```

Reply:
> It doesn't seem to work?
> 
> ```

Forgot that `run-webkit-tests` uses the older `optparse` instead of `argparse`. So never mind, this suggestion won't work.

Reply:
Yeah, it would be a nice cleanup to migrate that over but it's probably outside the scope of this PR.